packages/search-ui-analytics-plugin/README.md

```diff
@@ -0,0 +1,29 @@
+# search-ui-analytics-plugin
```

Comment:
We don't have to do this now, but perhaps in a future PR.

We should briefly explain what Behavioral Analytics is, and note that, for instance this is an experimental feature of Elastic that is not yet available.

Note that doing so will start sending events to the Elastic server, and maybe enumerate what those events are.

It would be also be super helpful to document somewhere in our docs something about "Analytics" and how you can use middleware to subscribe and handle all of these various events.

Reply:
Agreed. Definitely, documentation is a future task when analytics becomes publicly accessible. Il follow up with a PR to document it elsewhere for now.